core/modules/layout/layout.admin.inc

```diff
@@ -250,8 +250,9 @@ function layout_settings_form($form, &$form_state, Layout $layout) {
     '#required' => TRUE,
   );
 
-  // Get the list of layout template options.
-  $all_template_info = layout_get_layout_template_info();
+  // Get the list of layout template options. The list needs to be rebuilt (see
+  // https://github.com/backdrop/backdrop-issues/issues/984)
+  $all_template_info = layout_get_layout_template_info(NULL, $rebuild = TRUE);
```

Comment:
@docwilmot yes. I am refactoring to accept an additional one that defaults to `FALSE` and is backwards-compatible 😄 ...API addition.